src/Components/Aspire.Azure.Messaging.EventHubs/EventHubsComponent.cs

```diff
@@ -101,4 +103,47 @@ protected static string GetNamespaceFromSettings(AzureMessagingEventHubsSettings
                 $"the '{configurationSectionName}' configuration section.");
         }
     }
+    protected sealed class AzureEventHubHealthCheck : IHealthCheck, IAsyncDisposable
+    {
+        private readonly EventHubProducerClient _probe;
+
+        public AzureEventHubHealthCheck(TSettings settings)
+        {
+            var probeOptions = new EventHubProducerClientOptions
+            {
+                Identifier = $"AspireEventHubHealthCheck-{settings.EventHubName}",
+                RetryOptions = new EventHubsRetryOptions()
+                {
+                    Mode = EventHubsRetryMode.Exponential,
+                    MaximumRetries = 3,
+                    Delay = TimeSpan.FromMilliseconds(800),
```

Comment:
where is 800 from?

Reply:
I'll hoist this into Aspire:Settings :)

Reply:
@eerhardt I don't think there is a precedent for health check retry options. I found `MongoDBSettings.HealthCheckTimeout` only.

Would adding a `RetryOptions HealthCheckRetryOptions` property in `AzureMessagingEventHubsSettings` work here?